app/src/main/java/nerd/tuxmobil/fahrplan/congress/schedule/FahrplanFragment.kt

```diff
@@ -623,7 +623,7 @@ class FahrplanFragment : Fragment(), MenuProvider {
                 if (isNow) {
                     setBackgroundColor(ContextCompat.getColor(timeTextView.context, R.color.schedule_time_column_item_background_emphasized))
                 } else {
-                    setBackgroundResource(R.drawable.schedule_time_column_time_text_background_normal)
+                    setBackgroundColor(ContextCompat.getColor(timeTextView.context, R.color.schedule_time_column_item_background_normal))
```

Comment:
This can be a bit more DRY now:

``` kotlin
val color = if (isNow) R.color.schedule_time_column_item_background_emphasized else R.color.schedule_time_column_item_background_normal
setBackgroundColor(ContextCompat.getColor(timeTextView.context, color))
```